substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/jdk/JDKInitializationFeature.java

```diff
@@ -159,6 +159,8 @@ public void afterRegistration(AfterRegistrationAccess access) {
             rci.initializeAtBuildTime("apple.security", JDK_CLASS_REASON);
         }
 
+        rci.initializeAtBuildTime("sun.rmi.transport.GC", "Loaded an unneeded library (rmi) in static initializer.");
```

Comment:
I think there's also a policy for this in JmxCommonFeature that can be removed.

Reply:
Thanks for the catch! I missed that that for some reason.